SMAnalysis/schema-LD.json

```diff
@@ -0,0 +1,52 @@
+{
+  "$schema": "http://json-schema.org/schema#",
+  "schemaVersion": "0.0",
+  "$id": "https://raw.githubusercontent.com/smart-data-models/dataModel.SocialMedia/master/SMAnalysis/schema.json",
+  "title": "Social Media - SMAnalysis",
+  "description": "This entity contains a harmonised description of a generic SMAnalysis made for the Social Media domain. This entity is primarily associated with the process of analysis of Social Media applications' posts.",
+  "type": "object",
+  "allOf": [
+    {
+      "$ref": "https://smart-data-models.github.io/data-models/common-schema.json#/definitions/Location-Commons"
+    },
+    {
+      "properties": {
+	  "type": {
+          "type": "string",
+          "description": "Property. NGSI-LD Entity Type. It must be equal to SMAnalysis.",
+          "enum": [
+            "SMAnalysis"
+          ]
+        },
+	  "hasConfidence": {
+          "type": "number",
+          "description": "Property.  Model:'https://schema.org/Number'. Units:'No unit'. It represents the confidence of the analysis. All units are accepted in [CEFACT](https://www.unece.org/cefact.html) code."

```

Comment:
in case there are no units it is not recommended to use the Units:'' clause. It should be removed

Reply:
Done.